freedomofpress/securedrop-debian-packaging
# Adds libyaml based extension for PyYAML wheel

I did a rebuild of the PyYAML-5.4.1 wheel, which now contains the cython extension. This is what we are seeing in the CI as `libyaml-dev` is present in the system.j

The commit also adds `libyaml-0-2` to the dependency for `securedrop-proxy` package.

Related to #241

## How to test?

- `./scripts/install-deps`
- `./scripts/build-sync-wheels --clobber -p ../securedrop-proxy`

This should generate the same wheel (with the same sha256sum).

After this one can build a debian package for `securedrop-proxy` and try to install it in a `Debian Buster` vm/container to verify that it installs.
